chimney/src/main/scala/io/scalaland/chimney/internal/compiletime/derivation/transformer/rules/TransformSealedHierarchyToSealedHierarchyRuleModule.scala

```diff
@@ -16,11 +16,9 @@ private[compiletime] trait TransformSealedHierarchyToSealedHierarchyRuleModule {
       (Type[From], Type[To]) match {
         case (SealedHierarchy(Enum(fromElements)), SealedHierarchy(Enum(toElements))) =>
           mapEachSealedElementToAnotherSealedElement(fromElements, toElements)
-        case (SealedHierarchy(_), _) =>
-          DerivationResult.attemptNextRuleBecause(
-            s"Type ${Type.prettyPrint[From]} is a sealed/enum type but ${Type.prettyPrint[To]} is not"
-          )
-        case (_, SealedHierarchy(_)) =>
+        case (SealedHierarchy(Enum(fromElements)), _) if !ctx.config.areOverridesEmpty =>
+          mapEachSealedElementToAnotherSealedElement(fromElements, List.empty)
```

Comment:
my first approach was like this but without the `if !ctx.config.areOverridesEmpty`. Only 3 tests failed, mostly because error messages were different:
* io.scalaland.chimney.PartialTransformerStdLibTypesSpec
* io.scalaland.chimney.IssuesSpec
* io.scalaland.chimney.PartialTransformerIntegrationsSpec

For example the test `io.scalaland.chimney.IssuesSpec.fix issue #121` error message changed from
```
Chimney can't derive transformation from io.scalaland.chimney.IssuesSpec.Foo to io.scalaland.chimney.IssuesSpec.Bar

io.scalaland.chimney.IssuesSpec.Bar
  maybeString: scala.collection.immutable.Seq[java.lang.String] - can't derive transformation from maybeString: scala.Option[scala.collection.immutable.Set[java.lang.String]] in source type io.scalaland.chimney.IssuesSpec.Foo
  nested: io.scalaland.chimney.IssuesSpec.BarNested - can't derive transformation from nested: io.scalaland.chimney.IssuesSpec.FooNested in source type io.scalaland.chimney.IssuesSpec.Foo

scala.collection.immutable.Seq[java.lang.String]
  derivation from foo.maybeString: scala.Option[scala.collection.immutable.Set[java.lang.String]] to scala.collection.immutable.Seq[java.lang.String] is not supported in Chimney!

java.lang.String
  derivation from foo.nested.num: scala.Option[scala.Int] to java.lang.String is not supported in Chimney!

io.scalaland.chimney.IssuesSpec.BarNested
  num: java.lang.String - can't derive transformation from num: scala.Option[scala.Int] in source type io.scalaland.chimney.IssuesSpec.FooNested

Consult https://chimney.readthedocs.io for usage examples.
```

to 

```
Chimney can't derive transformation from io.scalaland.chimney.IssuesSpec.Foo to io.scalaland.chimney.IssuesSpec.Bar

java.lang.String
  derivation from none: scala.None to java.lang.String is not supported in Chimney!

scala.collection.immutable.Seq[java.lang.String]
  derivation from none: scala.None to scala.collection.immutable.Seq[java.lang.String] is not supported in Chimney!

io.scalaland.chimney.IssuesSpec.Bar
  maybeString: scala.collection.immutable.Seq[java.lang.String] - can't derive transformation from maybeString: scala.Option[scala.collection.immutable.Set[java.lang.String]] in source type io.scalaland.chimney.IssuesSpec.Foo
  nested: io.scalaland.chimney.IssuesSpec.BarNested - can't derive transformation from nested: io.scalaland.chimney.IssuesSpec.FooNested in source type io.scalaland.chimney.IssuesSpec.Foo

scala.collection.immutable.Seq[java.lang.String]
  can't transform coproduct instance scala.None to scala.collection.immutable.Seq[java.lang.String]
  can't transform coproduct instance scala.Some[scala.collection.immutable.Set[java.lang.String]] to scala.collection.immutable.Seq[java.lang.String]

java.lang.String
  derivation from some: scala.Some[scala.Int] to java.lang.String is not supported in Chimney!

java.lang.String
  can't transform coproduct instance scala.None to java.lang.String
  can't transform coproduct instance scala.Some[scala.Int] to java.lang.String

io.scalaland.chimney.IssuesSpec.BarNested
  num: java.lang.String - can't derive transformation from num: scala.Option[scala.Int] in source type io.scalaland.chimney.IssuesSpec.FooNested

scala.collection.immutable.Seq[java.lang.String]
  derivation from some: scala.Some[scala.collection.immutable.Set[java.lang.String]] to scala.collection.immutable.Seq[java.lang.String] is not supported in Chimney!

Consult https://chimney.readthedocs.io for usage examples.
```

which would make it worse...

Since when the target is not a sealed trait, the user needs to configure the transformation, adding the `!ctx.config.areOverridesEmpty` will only trigger this rule if some runtime overrides exists. WDYT?